app/Http/Controllers/Web/ImportFileController.php

```diff
@@ -174,13 +177,38 @@ public function store(Request $request, $importType)
             $importFile->import_type = $importType;
             $importFile->save();
 
+            $html = 'Created <a href="/import-files/'.$importFile->id.'">import #'.$importFile->id.'</a>.<ul>';
             $job = new ImportRockTheVoteRecord($row, $importFile);
+            $result = $job->handle();
```

Comment:
I'd recommend using Laravel's [`dispatchNow`](https://laravel.com/docs/5.7/queues#synchronous-dispatching) to dispatch this (rather than manually invoking the `handle` method). That way, we don't break dependency injection if we end up relying on other classes in this job down the line.



```suggestion
            $result = ImportRockTheVoteRecord::dispatchNow($row, $importFile);
```